src/pages/workspaces/workspace/Data.js

```diff
@@ -68,41 +69,48 @@ class WorkspaceData extends Component {
       ]),
     _.toPairs(workspaceEntities))
 
-    const entityTable = () => h(DataTable, {
-      defaultItemsPerPage: this.state.itemsPerPage,
-      onItemsPerPageChanged: itemsPerPage => this.setState({ itemsPerPage }),
-      initialPage: this.state.pageNumber,
-      onPageChanged: pageNumber => this.setState({ pageNumber }),
-      dataSource: _.sortBy('name', selectedEntities),
-      tableProps: {
-        rowKey: 'name',
-        scroll: { x: true },
-        columns: _.concat([
-          {
-            title: selectedEntityType + '_id',
-            key: selectedEntityType + '_id',
-            render: entity => entity.name
-          }
-        ], _.map(name => {
-          return {
-            title: name,
-            key: name,
-            render: entity => entity.attributes[name]
-          }
-        }, workspaceEntities[selectedEntityType]['attributeNames']))
-      }
-    })
+    const entityTable = () => h(Fragment, [
+      h(DataTable, {
+        defaultItemsPerPage: this.state.itemsPerPage,
+        onItemsPerPageChanged: itemsPerPage => this.setState({ itemsPerPage }),
+        initialPage: this.state.pageNumber,
+        onPageChanged: pageNumber => this.setState({ pageNumber }),
+        dataSource: _.sortBy('name', selectedEntities),
+        tableProps: {
+          rowKey: 'name',
+          scroll: { x: true },
+          columns: _.concat([
+            {
+              title: selectedEntityType + '_id',
+              key: selectedEntityType + '_id',
+              render: entity => entity.name
+            }
+          ], _.map(name => {
+            return {
+              title: name,
+              key: name,
+              render: entity => entity.attributes[name]
+            }
+          }, workspaceEntities[selectedEntityType]['attributeNames']))
+        }
+      }),
+      !freshEntities && spinnerOverlay
```

Comment:
I like that you're naming this according to the context, but I might stick with the `isFreshData` pattern everywhere to establish the pattern more clearly across the codebase.